adoc/Private-Registry-Installation.adoc

```diff
@@ -731,10 +737,10 @@ redis:
 . [[install-resource-limits]] (Optional) Setup Resource Requests and Limits
 +
 It is a good practice to specify resource requests and limit values.
-For each Harbor component it is possible to specify minimal resource value, i.e. the amount of CPU units and memory it should get, as well as a limit value so that Kubernetes knows the resources given to a component cannot exceed the such limit.
-These per-component values are used for all containers that are created for given Harbor component.
+For each Harbor component, it is possible to specify a minimal resource value, i.e. the amount of CPU units and memory it should get, as well as a limit value, so that Kubernetes knows the resources given to a component cannot exceed the limit.
```

Comment:
... value, i.e. the amount of CPU units and memory ...

Shouldn't we avoid abbreviations in general? 
... value, that is the amount of CPU units and memory ...

Following the style guide, we do not use "as well as" (if that did not change recently):
... it should get, and a limit value, so ...